packages/multiple-choice/configure/src/defaults.js

```diff
@@ -16,9 +16,14 @@ export default {
     rationaleEnabled: true,
     limitChoicesNumber: true,
     teacherInstructionsEnabled: true,
-    studentInstructionsEnabled: true
+    studentInstructionsEnabled: true,
+    verticalMode: true
   },
   configuration: {
+    verticalMode: {
+      settings: true,
```

Comment:
I would rather not display the toggle for Vertical Mode in the Settings Panel (should be `settings: false`).
By default, in the model, it should be true, that's correct.

Reply:
Also, we should add the new property in the docs. (pie-models)